reference-implementation/to-upstream-wpts/transform-streams/errors.js

```diff
@@ -308,4 +308,19 @@ promise_test(t => {
   });
 }, 'erroring during write with backpressure should result in the write failing');
 
+promise_test(t => {
+  const ts = new TransformStream({}, undefined, { highWaterMark: 0 });
+  return delay(0).then(() => {
+    const writer = ts.writable.getWriter();
+    // write should start synchronously
+    const writePromise = writer.write(0);
+    const abortPromise = writer.abort();
+    return promise_rejects(t, new TypeError(), ts.readable.getReader().read(), 'read() should reject')
```

Comment:
Why not one Promise.all covering all three at once?

Reply:
I think I was trying to make it clearer why the read() was there. But it didn't work, because I couldn't remember why it was needed. I have added comments instead.